app/test_helpers.go

```diff
@@ -115,9 +115,9 @@ func IntegrationTestNetworkConfig() network.Config {
 		LiquidationThreshold:   sdk.MustNewDecFromStr("0.05"),
 		BaseBorrowRate:         sdk.MustNewDecFromStr("0.02"),
 		KinkBorrowRate:         sdk.MustNewDecFromStr("0.2"),
-		MaxBorrowRate:          sdk.MustNewDecFromStr("1.5"),
+		MaxBorrowRate:          1_5000,
```

Comment:
for bigger numbers we can use `_` to separate "decimal". Note: we can't write `0_100` (for 0.1), because go would interpret it in octal notation.